list_and_dictionary.py

```diff
@@ -0,0 +1,105 @@
+lloyd = {
+  "name": "Lloyd",
+  "homework": [90.0,97.0,75.0,92.0],
+  "quizzes": [88.0,40.0,94.0],
+  "tests": [75.0,90.0]
+}
+alice = {
+  "name": "Alice",
+  "homework": [100.0, 92.0, 98.0, 100.0],
+  "quizzes": [82.0, 83.0, 91.0],
+  "tests": [89.0, 97.0]
+}
+tyler = {
+  "name": "Tyler",
+  "homework": [0.0, 87.0, 75.0, 22.0],
+  "quizzes": [0.0, 75.0, 78.0],
+  "tests": [100.0, 100.0]
+}
+
+## print the student's name
+
+student = [lloyd.get("name",),alice.get("name"),tyler.get("name")]
+
+print(student)
+
+## print the student's homework
+
+student = [lloyd.get("homework",),alice.get("homework"),tyler.get("homework")]
+
+print(student)
+
+## print the student's quizzes
+
+student = [lloyd.get("quizzes",),alice.get("quizzes"),tyler.get("quizzes")]
+
+print(student)
+
+## print the student's tests
+
+student = [lloyd.get("tests",),alice.get("tests"),tyler.get("tests")]
+
+print(student)
+
+###Define a function called average that has one argument, numbers.
+
+def average(numbers):
+    for item in numbers:
+        total = float (sum(numbers))
+        return total / len(numbers)
+
+
+##Write a function called get_average that takes a student dictionary (like lloyd, alice, or tyler) as input and returns his/her weighted average.
+
+def get_average(student):
+        homework = average(student.get("homework"))
+        quizzes = average(student.get("quizzes"))
+        tests = average(student.get("tests"))
+
+        homework = homework * 0.1
+        quizzes = quizzes * 0.3
+        tests = tests * 0.6
+        return homework + quizzes + tests
+  
+  
+### Define a new function called get_letter_grade that has one argument called score. Expect score to be a number.
+
+def get_letter_grade(score):
+	if score >= 90:return "A"
```

Comment:
I know this works too, but the better code style is to have `return` in a new line

```
if score >= 90:
    return "A"
elif score >= 80:
    return "B"